FileIO/MPI_IO/NodePartitionedMeshReader.cpp

```diff
@@ -71,21 +72,24 @@ MeshLib::NodePartitionedMesh* NodePartitionedMeshReader::read(
 
     MeshLib::NodePartitionedMesh* mesh = nullptr;
 
+    std::string confirmed_file_name_base = file_name_base;
+    boost::erase_last(confirmed_file_name_base, ".vtk");
```

Comment:
The name was changed. ``boost::erase_last_copy()`` needs an additional string variable as output. Since  boost::erase_last is only called once here, it is kept.